### PR TITLE
Feature/bai 178 integrate dark mode

### PR DESCRIPTION
**Pull request by ARADDCC002**

### CODE REVIEW

**Comment by a3957273** [comment]
Screenshot 2022-08-08 at 13 33 40

Screenshot 2022-08-08 at 13 34 12

Screenshot 2022-08-08 at 13 34 17

Hovering over buttons breaks things.

**Review by a3957273 — CHANGES_REQUESTED** [review]

File: pages/deployment/[uuid].tsx [hunk 1]
```diff
@@ -68,13 +71,7 @@ function CodeLine({ line }) {
...
         ${' '}
         <Tooltip title='Copy to clipboard' arrow>
-          <b
-            style={{
-              background: '#e0e0e0',
-            }}
-          >
-            {line}
-          </b>
+          <b>{line}</b>
```
[the remark this answers is not in the document]
Comment (ARADDCC002) [reply pages/deployment/[uuid].tsx:74]: Added a background to the text so it stands out, and a border radius so it somewhat resembles a button.

File: pages/model/[uuid].tsx [hunk 2]
```diff
@@ -16,13 +16,14 @@ import Stack from '@mui/material/Stack'
...
 import copy from 'copy-to-clipboard'
-import UploadIcon from '@mui/icons-material/Upload'
-import EditIcon from '@mui/icons-material/Edit'
-import PostAddIcon from '@mui/icons-material/PostAdd'
-import Favorite from '@mui/icons-material/Favorite'
+import UploadIcon from '@mui/icons-material/UploadTwoTone'
+import EditIcon from '@mui/icons-material/EditTwoTone'
+import PostAddIcon from '@mui/icons-material/PostAddTwoTone'
+import Favorite from '@mui/icons-material/FavoriteTwoTone'
 import DownArrow from '@mui/icons-material/KeyboardArrowDown'
```
[the remark this answers is not in the document]
Comment (ARADDCC002) [reply pages/model/[uuid].tsx:23]: Fixed

File: pages/review.tsx [hunk 3]
```diff
@@ -150,14 +177,17 @@ function ApprovalList({ type, category }: { type: RequestType; category: ReviewF
...
+                    <MuiLink
+                      variant='h5'
+                      sx={{ fontWeight: '500', textDecoration: 'none', color: theme.palette.secondary.main }}
+                    >
                       {requestObj.deployment?.metadata?.highLevelDetails?.name}
                     </MuiLink>
                   </Link>
                   <Typography variant='body1'>
                     Requesting deployment of{' '}
                     <Link href={`/model/${requestObj.deployment?.model?.uuid}`} passHref>
```
[the remark this answers is not in the document]
Comment (ARADDCC002) [reply pages/review.tsx:189]: As said in another comment I can't see us implementing our own Link component anywhere. Happy to do it all in this ticket but might be worth putting in its own one?

File: pages/index.tsx [hunk 4]
```diff
@@ -90,7 +103,13 @@ export default function ExploreModels() {
 
                 <Stack direction='row' spacing={1} sx={{ marginBottom: 2 }}>
                   {model.currentMetadata.highLevelDetails.tags.map((tag: string) => (
-                    <Chip color='primary' key={`chip-${tag}`} label={tag} size='small' variant='outlined' />
+                    <Chip
+                      sx={{ backgroundColor: '#f5f5f5', color: '#000000de' }}
```
[the remark this answers is not in the document]
Comment (ARADDCC002) [reply pages/index.tsx:107]: Think I've changed back all the ones except the model overview (just due to the background colour and outlined variant of the chip)

**Comment by ARADDCC002** [comment]
> Screenshot 2022-08-08 at 13 33 40
> 
> Screenshot 2022-08-08 at 13 34 12
> 
> Screenshot 2022-08-08 at 13 34 17
> 
> Hovering over buttons breaks things.

This should now be fixed. I don't know why but the white hover colour wasn't appearing for me..... but got someone else testing my new fix.

**Review by a3957273 — APPROVED** [review]

**Pull request merged**